.github/ISSUE_TEMPLATE/intent-to-ship--i2s-.md

```diff
@@ -51,8 +51,62 @@ Provide instructions for how to demo your feature such as a link to a demo page.
 Add any other information that may be relevant in determining if your feature can ship.
 -->
 
+## Build checklist
+
+This section describes considerations the implementor should keep in mind while building this technical solution.
+
+- [ ] Tests written
+<!--
+Have unit tests been written? Integration tests? Validator tests?
+Details on unit and integration tests:https://github.com/ampproject/amphtml/blob/master/contributing/getting-started-e2e.md#testing-your-changes
+Details on validator tests: https://github.com/ampproject/amphtml/blob/master/contributing/component-validator-rules.mdo
+-->
+- [ ] Documentation present
+<!--
+Has reference documentation been added? Have examples been added? Has an example been made available on amp.dev?
+-->
+- [ ] Cross platform testing
+<!--
+Have you tested all browsers and versions that AMP supports? https://amp.dev/support/faq/supported-browsers/
+Have you tested across mobile and desktop devices? 
+-->
+
+### Considerations checklist
+
+This section describes different cross cutting considerations that the implementor should keep in mind while building this technical solution.
+
+- [ ] Accessibility Considerations
+<!--
+Reach out to the UI and Accessibility WG for assistance. https://github.com/ampproject/wg-ui-and-a11y
```

Comment:
Is this a suggestion or a must?

Reply:
Added a comment above all the sections asking to only retain the sections are applicable. 

Also changed them from checkboxes to sub headings.

Reply:
I think it is really important to highlight when something is not applicable.

Reply:
Hey,

Sorry, I hadn't reviewed this again, because this comment wasn't addressed. I see above that you are asking sections to be removed. I don't think that is the right cause of action. Instead folks should mark a section as `n/a` possibly with a short sentence explaining why.

I also think the "Reach out" here is too affirmative. 

I'd write this as:

"Enter a short snippet for accessibility considerations of the design. If none apply, enter n/a including a short sentence if not obvious. If you need assistance, reach out…"